.github/workflows/ci.yml

```diff
@@ -17,12 +17,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby:
-          - 2.6
-          - 2.7
+        ruby: ["2.6", "2.7", "3.0"]
         build_types: ["LINT", "REDIS", "ACTIVERECORD"]
         exclude:
-          - ruby: 2.6
+          - ruby: "2.6"
```

Comment:
To avoid running lint multiple times this block should be changed to e.g.

```yml
        build_types: ["REDIS", "ACTIVERECORD"]
        include:
          - ruby: "3.0"
            build_types: "LINT"
```

Reply:
Ah nice, will try that :ok_hand: